stable/osg-frontier-squid/osg-frontier-squid/templates/configmap.yaml

```
@@ -0,0 +1,25 @@
+{{ if .Values.ExtraConf.Enabled }}
+apiVersion: apps/v1
```

```suggestion
apiVersion: v1
```

configmap isn't in apps/v1